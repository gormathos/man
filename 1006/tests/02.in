C1;C2;C3;C4;C5;C6;C7;C8;C9;C10;C11;C12;C13;C14;C15;C16;C17;C18;C19;C20;C21;C22;C23;C24;C25;C26;C27;C28;C29;C30;C31;C32;C33;C34;C35;C36;C37;C38;C39;C40
46;13;35;50;48;12;34;15;39;46;55;16;40;55;53;20;31;21;35;50;24;27;23;44;42;28;30;38;38;20;18;45;55;54;18;50;18;29;15;53
