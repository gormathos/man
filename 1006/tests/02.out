1383
55
12
